Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace BootCamp.Chapter
+{
+    /// <summary>
+    /// Part 1.
+    /// </summary>
+    public static class TextTable
+    {
+        /*
+
+         Input: "Hello", 0
+           +-----+
+           |Hello|
+           +-----+
+           
+         Input: $"Hello{Environment.NewLine}World!", 0
+           +------+
+           |Hello |
+           |World!|
+           +------+
+           
+         Input: "Hello", 1
+           +-------+
+           |       |
+           | Hello |
+           |       |
+           +-------+
+
+         */
+
+        /// <summary>
+        /// Build a table for given message with given padding.
+        /// Padding means how many spaces will a message be wrapped with.
+        /// Table itself is made of: "+-" symbols. 
+        /// </summary>
+        public static string Build(string message, int padding)
+        {
+            return Program.Frame(message, padding);
```

Comment:
This function isn't just for the looks or code to be testable. Things were supposed to be the other way round - implementation to be in here. In high levels (like the Program.cs class) you should only call other things, start the app and have nothing more. In other classes (Like this one) you should implement the meissing functionality.